chat-plugins/mafia.js

```diff
@@ -1029,6 +1037,28 @@ exports.commands = {
 			game.updateHost();
 		},
 
+		subhost: function (target, room, user) {
+			if (!this.canTalk()) return;
+			if (!target) return this.parse('/help mafia subhost');
+			this.splitTarget(target);
+			let targetUser = this.targetUser;
+			if (!targetUser || !targetUser.connected) return this.errorReply(`The user "${this.targetUsername}" was not found.`);
+			if (!room.users[targetUser.userid]) return this.errorReply(`${targetUser.name} is not in this room, and cannot be hosted.`);
+			if (!user.can('mute', null, room) && !user.can('broadcast', null, room)) return this.errorReply(`/mafia subhost - Access denied.`);
+			if (!user.can('mute', null, room) && targetUser.userid !== user.userid) return this.errorReply(`/mafia subhost - Access denied for subhosting to users other than yourself.`);
+			if (Object.keys(room.game.players).includes(targetUser.userid)) return this.errorReply(`You cannot subhost to a user in the game.`);
+			if (room.game.hostid === targetUser.userid) return this.errorReply(`${targetUser.name} is already the host.`);
+
+			if (Users(room.game.hostid)) Users(room.game.hostid).send(`>view-mafia-${room.id}\n|deinit`);
+			if (room.game.subs.includes(targetUser.userid)) room.game.subs.splice(room.game.subs.indexOf(targetUser.userid), 1);
+			room.game.host = targetUser.name;
+			room.game.hostid = targetUser.userid;
```

Comment:
You should probably push the host into room.game.played array (cause it would make sense that hosts can't sub back in the game)

Reply:
the host gets added automatically, new host probably should though